src/compiler/diagnosticMessages.json

```diff
@@ -1,5849 +1,5853 @@
+        "category": "Error",
+        "code": 2401
+    },
+    "Expression resolves to '_super' that compiler uses to capture base class reference.": {
+        "category": "Error",
+        "code": 2402
+    },
+    "Subsequent variable declarations must have the same type.  Variable '{0}' must be of type '{1}', but here has type '{2}'.": {
+        "category": "Error",
+        "code": 2403
+    },
+    "The left-hand side of a 'for...in' statement cannot use a type annotation.": {
+        "category": "Error",
+        "code": 2404
+    },
+    "The left-hand side of a 'for...in' statement must be of type 'string' or 'any'.": {
+        "category": "Error",
+        "code": 2405
+    },
+    "The left-hand side of a 'for...in' statement must be a variable or a property access.": {
+        "category": "Error",
+        "code": 2406
+    },
+    "The right-hand side of a 'for...in' statement must be of type 'any', an object type or a type parameter, but here has type '{0}'.": {
+        "category": "Error",
+        "code": 2407
+    },
+    "Setters cannot return a value.": {
+        "category": "Error",
+        "code": 2408
+    },
+    "Return type of constructor signature must be assignable to the instance type of the class.": {
+        "category": "Error",
+        "code": 2409
+    },
+    "The 'with' statement is not supported. All symbols in a 'with' block will have type 'any'.": {
+        "category": "Error",
+        "code": 2410
+    },
+    "Property '{0}' of type '{1}' is not assignable to string index type '{2}'.": {
+        "category": "Error",
+        "code": 2411
+    },
+    "Property '{0}' of type '{1}' is not assignable to numeric index type '{2}'.": {
+        "category": "Error",
+        "code": 2412
+    },
+    "Numeric index type '{0}' is not assignable to string index type '{1}'.": {
+        "category": "Error",
+        "code": 2413
+    },
+    "Class name cannot be '{0}'.": {
+        "category": "Error",
+        "code": 2414
+    },
+    "Class '{0}' incorrectly extends base class '{1}'.": {
+        "category": "Error",
+        "code": 2415
+    },
+    "Property '{0}' in type '{1}' is not assignable to the same property in base type '{2}'.": {
+        "category": "Error",
+        "code": 2416
+    },
+    "Class static side '{0}' incorrectly extends base class static side '{1}'.": {
+        "category": "Error",
+        "code": 2417
+    },
+    "Type of computed property's value is '{0}', which is not assignable to type '{1}'.": {
+        "category": "Error",
+        "code": 2418
+    },
+    "Class '{0}' incorrectly implements interface '{1}'.": {
+        "category": "Error",
+        "code": 2420
+    },
+    "A class can only implement an object type or intersection of object types with statically known members.": {
+        "category": "Error",
+        "code": 2422
+    },
+    "Class '{0}' defines instan
```

Comment:
looks like whitespace changed in this file. Can you undo that change please?

Reply:
That seems like a recurring problem, at least for me. I fixed this merge conflict through the GitHub UI and this is what I got. Do you know why this might be happening? (Windows line endings for example?)  I wonder if there's a way it can be resolved on master such that it stops happening.

In the meantime, I'll fix it on this branch shortly...

Reply:
What editor do you use? Do you have prettier or other formatting configured? The linux+emacs and macos+code people don't have this problem as far as I'm aware, although it does have Windows line endings.

Reply:
Hm, I guess it's possible that the problem *only* happens in the GitHub merge UI. I didn't keep careful track of when it happened and when it didn't. Anyway, fixed now :)